Pod/Classes/LSClockState.h

```diff
@@ -1,15 +1,15 @@
 #import <Foundation/Foundation.h>
 NS_ASSUME_NONNULL_BEGIN
 
-/// A straight port/copy of the `rl-cruntime-common` ClockState javascript prototype.
-@interface LSClockState : NSObject
+@protocol LSTimeProvider
+- (NSDate *)currentTime;
+- (SInt64)offsetInMicroseconds;
```

Comment:
I had imagined that the time provider would only have a `currentTime` method... `LSTracer` would either have a user-specified time provider XOR a `nil` time provider, and the clock correction code would all be behind `if (!self.timeProvider) { }` guards.